supervisor/coresys.py

```
@@ -107,6 +113,25 @@
         await self.config.read_data()
         return self
 
+    async def init_machine(self):
+        """Initialize machine information"""
```

Ruff (0.8.2)

First line should end with a period

Add period

(D400)